components/common/useRelativeTime.ts

```diff
@@ -31,7 +31,7 @@ export const getRelativeTime = (time: Date | number) => {
     const amount = Math.floor(diff / duration)
     return {
         formatted: rtf.format(amount, unit ),
-        updatesIn: duration - (Math.abs(diff - amount)),
+        updatesIn: 10 * minute,
```

Comment:
Maybe do a `Math.max` instead?

```suggestion
        updatesIn: Math.max(10 * minute, duration - (Math.abs(diff - amount))),
```